packages/modular-scripts/src/build/index.ts

```diff
@@ -135,35 +133,25 @@ async function buildStandalone(
     // create-react-app doesn't support plain module outputs yet,
     // so --preserve-modules has no effect here
 
-    const buildScript = require.resolve(
-      'modular-scripts/react-scripts/scripts/build.js',
-    );
+    // If it's an app, set it at ESBUILD_TARGET_FACTORY or default to es2015
+    // If it's not an app it's an ESM view, then we need es2020
+    const esbuildTargetFactory = isApp ? browserTarget : ['es2020'];
+    let targetPath = await getWorkspaceLocation(target);
 
-    // TODO: this shouldn't be sync
```

Comment:
We no longer run the .js file separately with ENV variables, instead calling a simple buildWebpack() function, passing it all the necessary variables.